src/Compilers/CSharp/Test/Semantic/Semantics/OverloadResolutionPerfTests.cs

```diff
@@ -551,5 +550,72 @@ public void NullableStateManyNestedFunctions()
                 //         return value;
                 Diagnostic(ErrorCode.WRN_NullReferenceReturn, "value").WithLocation(16395, 16));
         }
+
+        [WorkItem(51739, "https://github.com/dotnet/roslyn/issues/51739")]
+        [ConditionalFact(typeof(WindowsOnly), typeof(IsRelease))]
```

Comment:
I feel like these tests could perform their function portably if the nesting level and max depth were, say, 20 and 10 respectively. I would feel a little better about this as it seems like someone inadvertently breaking the Enter/ExitWalker calls will cause this test to start failing.